vllm/v1/sample/sampler.py

```diff
@@ -181,14 +183,17 @@ def apply_penalties(
             apply_min_token_penalties(logits,
                                       sampling_metadata.output_token_ids,
                                       sampling_metadata.min_tokens)
+
         if not sampling_metadata.no_penalties:
             assert sampling_metadata.prompt_token_ids is not None
             logits = apply_all_penalties(
-                logits, sampling_metadata.prompt_token_ids,
+                logits,
+                sampling_metadata.prompt_token_ids,
                 sampling_metadata.presence_penalties,
                 sampling_metadata.frequency_penalties,
                 sampling_metadata.repetition_penalties,
-                sampling_metadata.output_token_ids)
+                sampling_metadata.output_token_ids,
+            )
```

Comment:
Same general comment, it's best not to include unrelated formatting changes, looks like there are a couple above and some in the other file.

Reply:
yeah, it's done by my VSCode, sorry.

I will be more cautious next time.

Reply:
okay, confirmed, actually it's done by the yapf in the pre-commit hooks. It forced me to do such change...